components/menu/test/menu.test.js

```diff
@@ -117,6 +117,14 @@ describe('d2l-menu', () => {
 			expect(document.activeElement).to.equal(elem.querySelector('#c1'));
 		});
 
+		it('sets focus to next item that starts with uppercase character pressed', () => {
+			const eventObj = document.createEvent('Events');
+			eventObj.initEvent('keypress', true, true);
+			eventObj.charCode = 67;
```

Comment:
Hopefully we can kill these legacy events and just use `new KeyboardEvent()` and `new Event()` soon now that IE11 is officially dead.